packages/xo-web/src/common/xo/index.js

```diff
@@ -965,28 +968,40 @@ export const startVm = async (vm, host) => {
 
   const id = resolveId(vm)
   const hostId = resolveId(host)
-  return _call('vm.start', {
-    id,
-    host: hostId,
-  }).catch(async reason => {
-    if (!forbiddenOperation.is(reason)) {
+  return _startVm(id, hostId).catch(async reason => {
+    if (!forbiddenOperation.is(reason) && !operationFailed.is(reason)) {
```

Comment:
You still need to check the error code, though, since an `operationFailed` error isn't necessarily due to duplicated MAC addresses.